inst/WORDLIST

```diff
@@ -60,21 +64,19 @@ https
 IG
 knitr
 labelled
+linter
+LLC
 MedDRA
+metacore
 metatools
-Metatools
 mmHg
 modularized
 Mosteller
 msec
+na
 OCCDS
-occuring
-param
-Param
-PARAM
+optionality
 paramater
```

Comment:
We shouldn't allow misspellings of words to come through - this is a misspelling in 
![image](https://user-images.githubusercontent.com/10111024/154859780-f28e282d-0d79-4071-934e-e8b065b66a14.png).  Could you make this into an issue for someone to update.   This is a good one for someone new to admiral to fix.

Reply:
Should we really fix mistakes in a separate issue? I wonder if it would be easier to do both updating the list and fixing mistakes in this issue.